tests/cmd/main_test.py

```diff
@@ -6,6 +6,7 @@
 import os.path
 
 import mock
+from six import StringIO
```

Comment:
imo use six, it's what it's for

But in this case you can avoid it entirely, I'll comment below :D